src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -3,72 +3,130 @@
 using System.IO;
 using System.Linq;
 using System.Text;
+using System.Threading.Tasks;
 using Microsoft.DotNet.ProjectModel;
```

Comment:
Out of scope for this PR, but have you looked into doing the NuGet math without `Microsoft.DotNet.ProjectModel` APIs? They were deprecated along with the rest of project.json